### PR TITLE
Initial draft on project description and purpose

### PR DESCRIPTION
**Pull request by johanlinaker**
My initial take on the goal and purpose of the project. May serve as a foundation for further discussion and development.

### CODE REVIEW

**Review by altaired — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+
+Currently we are working with the use case of visualizing available employment opportunities in collaboration with the Swedish Employment Agency (Arbetsförmedlingen). This is also what triggered the creation of data-universe at an OpenHack event, which is a hackathon focused on addressing humanitarian and social challenges. 
+
+In the future we aim to broaden the scope to include further use cases, in order to make the world of data more accessible and understandable. 
 
 ## Setup
 
+The project is currently in an early state why it may not be ready to most end-users. However, if you have the technical background needed, and interested in setting it up, we encourage you to download the project and try it out! It is based on the D3.js framework and currently only tested on a Samsung mobile handset device with Virtual Reality capabilities.
```
Comment (marcusrettig) [inline README.md]: The framework we use is three.js, not D3.js. Also it was tested on an iPhone.

**Review by marcusrettig — CHANGES_REQUESTED** [review]
Overall good job! See comment about changes that need to be made under ## Setup section.

**Comment by johanlinaker** [comment]
Updated the readme accordingly, please verify changes.

**Comment by funrep** [comment]
Looks good! I'll just wait for Marcus to confirm he's OK having his email in the README.

**Comment by johanlinaker** [comment]
An alternative is that we buy a domain and add forwarding to your email(s). We should have this discussion outside of this PR though.

**Review by funrep — COMMENTED** [review]
Good, ready to merge!

**Review by funrep — APPROVED** [review]

**Review by marcusrettig — APPROVED** [review]

**Pull request merged**